### PR TITLE
feat: Adding API v2 Bigquery samples

### PR DESCRIPTION
**Pull request by thokalavinod**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ ] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [ ] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: .github/workflows/security-center-snippets-v2.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Google LLC
```
Comment (iennae) [inline .github/workflows/security-center-snippets-v2.yaml:1]: Is there a reason why a second security-center-snippets workflow needs to be created versus just updating the paths in the current one here? https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/workflows/security-center-snippets.yaml

Comment (thokalavinod) [reply .github/workflows/security-center-snippets-v2.yaml]: addressed

**Comment by iennae** [comment]
@gemini review

**Comment by subfuzion** [comment]
@thokalavinod I resubmitted your PR from a local branch so that workflow tests can run (they won't run for forks): #3919. I'm going to close this PR now. Thanks!

**Comment by subfuzion** [comment]
Obsoleted by #3847

**Pull request closed without merging**